.github/workflows/osv-auto-fix.yml

```
@@ -33,18 +33,23 @@ jobs:
           npm audit fix
-          npm update brace-expansion
 
           # Fix apps/web if present
           if [ -d "apps/web" ]; then
             npm --prefix apps/web install --ignore-scripts
             npm --prefix apps/web audit fix
-            npm --prefix apps/web update brace-expansion
           fi
 
+      - name: Ensure PR token is configured
+        if: ${{ !secrets.OSV_AUTO_FIX_TOKEN && !secrets.SYNC_PR_TOKEN }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)